server/mailet/integration-testing/src/test/java/org/apache/james/mailets/RemoteDeliveryErrorTest.java

```diff
@@ -68,64 +73,44 @@
 
     private static final String FROM = "from@" + DEFAULT_DOMAIN;
     private static final String RECIPIENT = "touser@" + ANOTHER_DOMAIN;
-    private static final String ALWAYS_421_RCPT_BEHAVIOR = "[{" +
```

Comment:
I think it comes from an earlier commit, rewrite them if so

Reply:
Sorry, at that point of the Sprint, I will not fix the history.